src/docfx/build/toc/TableOfContentsMap.cs

```diff
@@ -239,10 +242,11 @@ private void SplitToc(FilePath file, TableOfContentsNode toc, ConcurrentBag<File
                 }
 
                 var newNodeFilePath = new PathString(Path.Combine(Path.GetDirectoryName(file.Path) ?? "", $"_splitted/{name}/TOC.yml"));
-                var newNodeFile = FilePath.Generated(newNodeFilePath);
+                var newNodeFile = FilePath.Generated(newNodeFilePath, file);
 
                 _input.AddGeneratedContent(newNodeFile, new JArray { newNodeToken });
                 result.Add(newNodeFile);
+                _metadataProvider.GetMetadata(_errors, newNodeFile);
```

Comment:
this is not needed

Reply:
Actually it's needed. I use this to catch the result once the split toc files generated. Or it will cause some error. I will locate the problem later.